tests/unit/GithubContext/Application/EventListener/PullRequestInspectionStatusListenerTest.php

```diff
@@ -197,6 +197,22 @@ public function testItDoesNothingIfTheRepositoryDisabledTheInspections()
         $this->listener->onInspectionStarted($event);
     }
 
+    public function testItDoesNothingIfTheRepositoryIsInFlightMode()
+    {
+        $inspection = $this->createMock(Entity\PullRequestInspection::class);
+        $repository = $this->createMock(Entity\Repository::class);
+        $domainEvent = new Event\InspectionStarted($inspection);
```

Comment:
Parameter #1 $inspection of class Regis\GithubContext\Application\Event\InspectionStarted constructor expects Regis\GithubContext\Domain\Entity\PullRequestInspection, PHPUnit_Framework_MockObject_MockObject given.